### PR TITLE
Diverses upgrades

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v8.15.1
```
Comment (VincentCATILLON) [inline .nvmrc]: Why not only 8 ? It should be enough ans follow patches upgrades

Comment (AdrieanKhisbe) [reply .nvmrc]: C'est pour le dev du package. et notamment éviter les problème de version de npm. (d'où la version de 8 assez haute)

Comment (VincentCATILLON) [reply .nvmrc]: Yes but you can only define `8` in your `.nvmrc` file, it will follow upgrades :)

Comment (AdrieanKhisbe) [reply .nvmrc]: but it only consider the version you have installed

**Pull request merged**